examples/Toast.hasCloseButton.jsx

```diff
@@ -0,0 +1,24 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+import * as React from 'react';
+import { useToaster, Button } from '@itwin/itwinui-react';
+
+export default () => {
+  const toaster = useToaster();
+
+  React.useEffect(() => {
+    toaster.setSettings({
+      hasCloseButton: true,
+    });
```

Comment:
This does not look right.

![](https://github.com/user-attachments/assets/56e1f167-a68e-491d-b305-f553c20ebe16)


Isn't `hasCloseButton` an individual toast-level option, rather than a toaster-level setting?

Reply:
Right, it should have been set in the status method. I have made the fix.